modules/oxide-components/src/test/ts/browser/keynav/TabbingType.spec.tsx

```
@@ -0,0 +1,290 @@
+      const innerRefs = useMemo(() => new Array(3).fill(null).map(() => useRef<HTMLDivElement>(null)), []);
+      useTabKeyNavigation({
+        containerRef: outerRef,
+        selector: '.inner-level',
+        execute: (focused) => {
+          Focus.focus(SelectorFind.descendant<HTMLElement>(focused, '.item').getOrDie());
+          return Optional.some(true);
+        },
+      });
+      innerRefs.forEach((ref) => {
+        // eslint-disable-next-line react-hooks/rules-of-hooks
+        useTabKeyNavigation({
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)